README.md

```diff
@@ -1,28 +1,78 @@
 # github-action-detect-unused-secrets
 
-GitHub Action that automatically detects unused secrets comparing the workflow secret names with the repo secret names.
+GitHub Action that detects unused secrets.
+Compares the secret names in your repository with the secret names under `.github/workflows`
 
 ## Example usage
 
-```yaml
-name: detect-unused-secrets
+The example below runs the action every Monday at 10am.
+Generates a token with required permissions which `secrets.GITHUB_TOKEN` does not have.
+Uses checkout code to get access to the `$GITHUB_WORKSPACE`,
```

Comment:
this is an implementation detail which we don't need to clarify here. we can simply clarify that we need to check out the code so we have access to the content of the workflow files